<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>621</width>
     <height>33</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>621</width>
     <height>0</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>31</y>
     <width>621</width>
     <height>269</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelLocalPathIso">
      <property name="toolTip">
       <string>local path of our ISOs images</string>
      </property>
      <property name="text">
       <string>Local path ISOs:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEditLocalPathIso">
      <property name="toolTip">
       <string>local path of our ISOs images</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelLocalPathDeb">
      <property name="toolTip">
       <string>local path of our DEBS packages</string>
      </property>
      <property name="text">
       <string>Local path packages .deb:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEditLocalPathDeb">
      <property name="toolTip">
       <string>local path of our DEBS packages</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelRemoteHost">
      <property name="toolTip">
       <string>ip address of remote host where we want to export our ISOs and packages</string>
      </property>
      <property name="text">
       <string>Remote host:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="lineEditRemoteHost">
      <property name="toolTip">
       <string>ip address of remote host where we want to export our ISOs and packages</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelRemoteUser">
      <property name="toolTip">
       <string>user name to the uset to connect remote host</string>
      </property>
      <property name="text">
       <string>Remote user:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lineEditRemoteUser">
      <property name="toolTip">
       <string>user name to the uset to connect remote host</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelRemotePathIso">
      <property name="toolTip">
       <string>path on remote host where we store our ISOs files</string>
      </property>
      <property name="text">
       <string>Remote path ISOs:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lineEditRemotePathIso">
      <property name="toolTip">
       <string>path on remote host where we store our ISOs files</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="labelRemotePathDeb">
      <property name="toolTip">
       <string>path on remote host where we store our DEBS packages</string>
      </property>
      <property name="text">
       <string>Remote path packages deb:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="lineEditRemotePathDeb">
      <property name="toolTip">
       <string>path on remote host where we store our DEBS packages</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelFilterDeb">
      <property name="toolTip">
       <string>path on remote host where we store our DEBS packages</string>
      </property>
      <property name="text">
       <string>Filter packages deb:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditFilterDeb">
      <property name="toolTip">
       <string>path on remote host where we store our DEBS packages</string>
      </property>
      <property name="placeholderText">
       <string>eggs_9.*.*_ </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
